core/server/services/posts/posts-service.js

```diff
@@ -65,6 +76,14 @@ class PostsService {
             const sendEmail = model.wasChanged() && this.shouldSendEmail(model.get('status'), model.previous('status'));
 
             if (sendEmail) {
+                // Set the newsletter_id if it isn't passed to the API
+                if (!frame.options.newsletter_id) {
+                    const newsletters = await this.models.Newsletter.findAll({status: 'active'}, {transacting: frame.options.transacting});
```

Comment:
Super minor but I think we can add a `limit:1` to this so we don't load all of the newsletters

Reply:
I always have to look into the code to remember how to use things like `limit` with our models. Looking at https://github.com/TryGhost/Ghost/blob/a2b3568956477b995553e6662a71810e5a656952/core/server/models/base/plugins/sanitize.js#L45-L48 it seems that we don't support `limit` for the `findAll` method?

Reply:
Oh I need to switch to `findPage`, the naming does make sense 😅